test/pg_regress.sh

```diff
@@ -85,7 +85,7 @@ elif [[ -z ${TESTS} && ( -n ${SKIPS} || -n ${IGNORES} ) ]]; then
 
   echo > ${TEMP_SCHEDULE}
 
-  ALL_TESTS=$(grep '^test: ' ${TEST_SCHEDULE} 2>/dev/null | sed -e 's!^test: !!' |tr '\n' ' ')
+  ALL_TESTS=$(grep -a '^test: ' ${TEST_SCHEDULE} | sed -e 's!^test: !!' |tr '\n' ' ')
```

Comment:
Let's do it for the test output files as well? No point in failing the test when grep thinks the output is binary.

Reply:
Just did. :)

First didn't think it was very important, but it's better to dig through those files anyway since there is no harm.